test/test_componentdefinition.py

```diff
@@ -62,7 +62,6 @@ def testCDDisplayId(self):
         self.assertCountEqual(actual, expected)
 
     # See Issue #64, CD.assemblePrimaryStructure is not implemented
```

Comment:
This comment is probably obsolete and should be deleted

Reply:
Removed in d52c566